sky/compositor/compositor_options.cc

```diff
@@ -0,0 +1,43 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/compositor/compositor_options.h"
+
+#include <mutex>
+
+namespace sky {
+namespace compositor {
+
+CompositorOptions& CompositorOptions::Shared() {
+  static std::once_flag once;
+  static CompositorOptions* options = nullptr;
+  std::call_once(once, []() { options = new CompositorOptions(); });
```

Comment:
aside: sky currently compiles with -fno-threadsafe-statics (i.e. no magic statics) since that's what chromium did. do you plan to keep it this way? i've considered flipping this option the other way for mojo as it simplifies a bunch of things (i.e. you don't need to do this dance)